lms/djangoapps/certificates/admin.py

```diff
@@ -64,8 +64,9 @@ class CertificateGenerationCourseSettingAdmin(admin.ModelAdmin):
     """
     Django admin customizations for CertificateGenerationCourseSetting model
     """
-    list_display = ('course_key', 'enabled')
+    list_display = ('course_key', 'self_generation_enabled', 'language_specific_templates_enabled')
     search_fields = ('course_key',)
+    exclude = ('enabled',)
```

Comment:
+1, since this might be useful while transitioning.

Reply:
If you do not exclude it, does that mean it will be available in the form for people to set? If so, I think it would be safer to just exclude it as it reduces the likelihood that someone might be saving to that column while our migration to copy the data to the new field is running.

Reply:
Excluding this field from the admin form cuts off your easiest option for manual fixes should something go wrong. The best way to ensure folks don't use the field is to (a) note the deprecation in the help text and (b) communicate to those who regularly access the admin explaining the deprecation.